pages/index.js

```diff
@@ -275,16 +284,26 @@ export default function Home({
           <div className="mt-16 mb-8 space-y-16">
             {featuresDetails.map((feature, featureIdx) => (
               <div
+                ref={featureRefs[featureIdx][0]}
                 key={feature.name}
-                className="flex flex-col-reverse lg:grid lg:grid-cols-12 lg:items-center lg:gap-x-8"
+                className="flex flex-col-reverse overflow-x-hidden lg:grid lg:grid-cols-12 lg:items-center lg:gap-x-8"
               >
                 <div
-                  className={classNames(
+                  className={`
```

Comment:
Using the function `classNames` makes it much easier to read

Reply:
Ok, Done!

Reply:
Thank you but this was not used correctly, please see examples in the code base. I have added additional inline comments with a link